sycl/include/sycl/detail/properties_traits.def

```diff
@@ -17,3 +17,4 @@ __SYCL_PARAM_TRAITS_SPEC(sycl::property::reduction::initialize_to_identity)
 __SYCL_PARAM_TRAITS_SPEC(sycl::ext::oneapi::property::queue::priority_low)
 __SYCL_PARAM_TRAITS_SPEC(sycl::ext::oneapi::property::queue::priority_high)
 __SYCL_PARAM_TRAITS_SPEC(sycl::ext::oneapi::property::queue::priority_normal)
+__SYCL_PARAM_TRAITS_SPEC(sycl::property::buffer::mem_channel)
```

Comment:
I tried to find it in spec https://registry.khronos.org/SYCL/specs/sycl-2020/html/sycl-2020.html but failed. Where is it specified?

Reply:
It's an extension property that sadly doesn't follow the extension naming: https://github.com/steffenlarsen/llvm/blob/steffen/fix_mem_channel_trait_spec/sycl/doc/extensions/supported/sycl_ext_intel_mem_channel_property.asciidoc